configure.ac

```diff
@@ -1204,6 +1204,23 @@ AC_ARG_WITH(
     ]
 )
 
+AC_SUBST(systemd_unit_dir)
+AC_ARG_WITH(
+    systemd-unit-dir,
+    AS_HELP_STRING(
+	    [--with-systemd-unit-dir=DIR],
+	    [systemd unit directory @<:@SYSTEMD_UNIT_DIR@:>@
+        Defaults to /etc/systemd/system/]
+    ),
+    [
+	    systemd_unit_dir="$withval"
+    ],
+    [
+        # default to the system admin unit directory
+        systemd_unit_dir="/etc/systemd/system/"
```

Comment:
On Red Hat-based distros, `pkg-config systemd --variable=systemdsystemunitdir` works (it returns "`/usr/lib/systemd/system`").

Does that work on SUSE too? If so, we could use it as the default value here.